lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -334,11 +334,7 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
             );
     }
 
-    async cancelStartProcess() {
-        if (this.currentCreatedProcess) {
-            await this.startProcessCloudService.deleteProcess(this.appName, this.currentCreatedProcess.id);
-        }
-
+    cancelStartProcess() {
```

Comment:
don't we need unit test? 

I think also the name of the methods does't reflect the current behaviour. As far as I remember it was because of the separation we did ( and reverted ) between started process but not initialized? cc @VitoAlbano

Reply:
This is a two parts task, i expected to see some unit test removed though (or added in case we don't have. For the method name i'm fine with the actual one, makes enough sense